src/App.tsx

```diff
@@ -12,155 +15,38 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink
+            className={({ isActive }) =>
+              cn('navbar-item', { 'has-background-grey-lighter': isActive })
```

Comment:
Consider extracting the className logic into a separate function outside of the JSX to improve readability and potential reusability.